### ISSUE TITLE
Pre-populated database not imported properly

### ISSUE THREAD

**Issue by Stophface** [andpor/react-native-sqlite-storage#468]
## Expected Behavior
I expect the database to be moved and have the table in it 

## Current Behavior
I move the database and try to `SELECT` something from the table, but it returns that the table does not exist. 

## Possible Solution
I suspect the moving of the `database` has not worked. `SQLite` rather creates a completly new `database` on `SQLite.openDatabase({ name: 'foo', location: 'default' }...`, and of course, that has no `table: bar` yet. 

## Steps to Reproduce (for bugs)
I have a pre-populated `database` named `foo.db` located in `www/sqlite/`. I move this database to the internal storage like this

    try {
        await SQLite.openDatabase({ name: foo, createFromLocation: '~/sqlite/foo.db' });
        console.log('Database successfully moved.');
    } catch (error) {
        console.log('Error on moving database. Error: ', error);
    }

This logs me `Database successfully moved.` so I suspect it worked. 
Then I try to open the database like this.

    SQLite.openDatabase({ name: 'foo', location: 'default' },
        db => {
            console.log('Successfully opened Database:', db);
            db.transaction(tx => {
                tx.executeSql('SELECT * FROM bar', [], (_, { rows }) =>
                    console.log(JSON.stringify(rows))
                )
            });
        },
        error => {
            console.log('Error on opening database locations.db: ', error);
        });

This logs me `Successfully opened Database` with this

    {
        "dbname": "locations", 
        "openError": [Function anonymous], 
        "openSuccess": [Function anonymous], 
        "openargs": {
            "dblocation": "nosync", 
            "location": "default", 
            "name": "foo"
            }
    }

That looks like a `Database Object` to me. 
But then returns me on `tx.executeSql('SELECT * FROM bar'...` this

> Possible Unhandled Promise Rejection (id: 0):

    Object {
      "code": 5,
      "message": "no such table: bar",
    }

## Your Environment

* React Native SQLite Storage Version used: 5.0.0
* React Native version used: 0.63.4
* Operating System and version (simulator or device): iPhone 11 Pro Max 14.4

**Comment by adamgins** [comment]
@Stophface, did you make any progress with this?

**Comment by huzeyfetas** [comment]
The same issue... 

I have the same problem and have been looking for a solution for about a week. I'm downloading a database from the web service, for example, let's say hello.db. After moving it to the databases folder on the android side very comfortably, I took care of all my problems with the explanations in the documentation, it is superb, but I faced a complete disaster on the ios side. There is neither a databases folder that needs to be moved nor the opendatabase command can not read the database file I have. object {code 5, message: no such table table_name}

**Comment by infinitbility** [comment]
When you want pre-populate database then every time you have to create a new database file and delete previous database or re build application.

check out below documentation 

https://infinitbility.com/react-native-sqlite-storage-examples-of-query

**Comment by pentolbakso** [comment]
This is working for me. 
First time it didn't work.. but after delete the app and rebuilding. It works (I'm on IOS simulator)

```javascript
  console.log("opening db");
  const db = await SQLite.openDatabase({ name: "hadits.db", location: "default" });
  console.log("db opened");
  const result = await db.executeSql("SELECT * FROM hadits ORDER BY 'position' ASC");
  console.log("table loaded");
```

**Comment by osupa** [comment]
For me, there was a little bit of subtlety getting this to work on iOS as I kept experiencing the infamous "no such table found" error. I assumed that all I had to do was create a www directory under my project through the file system and it would all work. I created the directory and placed my sqlite db there and verified it in the project viewer in Xcode. However, it did not copy my database into /Users..../Library/Developer/CoreSimulator/Devices/MyApp.app/HASH-ID/Library/LocalDatabase/my-sqlite-db.db 

I only discovered the source of this issue when I used Xcode to step through the source code of 
react-native-sqlite-storage/platforms/ios/SQLite.m 
around line #159 which starts with

RCT_EXPORT_METHOD(open: (NSDictionary *) options success:(RCTResponseSenderBlock)success error:(RCTResponseSenderBlock)error)


"Step 3 - Add file to project
in XCode, right click on the main folder and select Add Files to 'your project name"

in https://github.com/andpor/react-native-sqlite-storage is very important. Use Xcode to add the folder.


Good luck.
Osupa.
